services/opencollective/opencollective.service.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const OpencollectiveBase = require('./opencollective-base')
+
+module.exports = class Opencollective extends OpencollectiveBase {
+  static get examples() {
+    return [
+      {
+        title: 'open collective backers and sponsors',
+        namedParams: { collective: 'shields' },
+        staticExample: this.render(35),
+        keywords: ['opencollective'],
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'backers and sponsors',
+    }
+  }
+
+  static get route() {
+    return this.buildRoute()
```

Comment:
I think this route probably needs to have something in it. How about `all`?

Could you also please rename file and its tester with `all` in the name? Usually when we have a file like `opencollective/opencollective.service.js` it's for a badge that hasn't been refactored yet, or because there is only one `.service.js` file for that service. Same with the testers. So it's confusing to have these two files here, without the qualifier.